test/adapter/json_api/parse_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        module Deserialization
+          class ParseTest < Minitest::Test
+            def setup
+              @hash = {
+                'data' => {
+                  'type' => 'photos',
+                  'id' => 'zorglub',
```

Comment:
But that only means the server expects single-resource requests (without the bulk extension)

The client would still need to handle multiple-resource responses